dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/IdentifiableObjectUtils.java

```diff
@@ -376,7 +375,7 @@ public static String getDisplayName( Object object )
         {
             return "[ object is null ]";
         }
-        else if ( IdentifiableObject.class.isInstance( object ) )
+        else if ( object instanceof IdentifiableObject )
```

Comment:
I think this should be isInstance( object ) and not instanceof.

Reply:
Fixed